### ISSUE TITLE
Strange Bug on ad_content

### ISSUE THREAD

**Issue by Kr3m** [cartman300/bspconv#5]
This also seems to happen with q3map2, but I'm not sure why.  It didn't always happen...  Every last ad_content banner looks like this now.

![shot-20240504-020920](https://github.com/sbarisic/bspconv/assets/290740/33af63d9-1129-45e0-ae8f-8680ad9ca163)

**Comment by sbarisic** [comment]
@Kr3m

This happens when converting a QL map to Q3, right?

**Comment by Kr3m** [comment]
@sbarisic Yes

**Comment by sbarisic** [comment]
That's because Quake 3 doesn't support the advertisement lump that Quake Live added. It's just a quad with a defined position and advertisement display model.

I could probably convert these to static meshes with a texture and embed them in the BSP, but that will take a while

**Comment by Kr3m** [comment]
I was able to get them working by subbing the "advertisement" classname with "func_static" when decompiling and recompiling a map, but of course the rest of the map is broken when doing that.

**Comment by Kr3m** [comment]
Okay so I'm not sure if this will help you or not, but I've found if I load the BSP file into Quark, and copy all of the entities to Notepad, I can do a search and replace to replace all occurrences of "advertisement" with "func_static" and then I have to remove all lines that start with cellid.  Once this is done, and I save the BSP, the ad_content works as expected.

**Comment by sbarisic** [comment]
> Okay so I'm not sure if this will help you or not, but I've found if I load the BSP file into Quark, and copy all of the entities to Notepad, I can do a search and replace to replace all occurrences of "advertisement" with "func_static" and then I have to remove all lines that start with cellid. Once this is done, and I save the BSP, the ad_content works as expected.

Can you provide me two sample BSP files? One with the original advertisement lump, and another one with replaced func_static?

**Comment by sbarisic** [comment]
Actually scrap that, i managed to get it working. I will upload a new release soon.

Here's a screenshot from bloodrun.bsp

![image](https://github.com/sbarisic/bspconv/assets/3836060/9979c9fd-b278-4b76-bc91-39ad3b0055ff)

**Comment by sbarisic** [comment]
Fixed in the latest release, if a problem or bug persists, please open a new issue
